### ISSUE TITLE
Script not launching anymore

### ISSUE THREAD

**Issue by BartiemanBro** [TWOverflow/twoverflow.github.io#2]
The script completly stopped working since the 1.91 update that was implemented yesterday
Please help :)

**Comment by ivanthz** [comment]
This is the bug:
Uncaught TypeError: Cannot read property 'isInitialized' of undefined

**Comment by BartiemanBro** [comment]
Well, do you have an idea on how to fix it? xD

**Comment by ivanthz** [comment]
We have to wait for mafrazz see what happens

**Comment by innogames-tw2** [comment]
Good luck 😉

**Comment by BartiemanBro** [comment]
In mafrazz we trust

**Comment by BartiemanBro** [comment]
It will be very interesting to see what happens next :)
No hate on you inno, your just keeping your business (and your money) safe, pretty normal thing.
But I'm not sure you'll be able to win this battle. The fact that you needed to adapt your code because of one smart guy, despite all the money you have is already impressing, Mr. Mafrazz has a good reason to be proud already. I'm sure he will accept the challenge.
The script is easier and faster to modify and adapt, especially in this case, it would maybe be easier for you if you developped the game the same way that TW1...
So, good luck to you too :D

**Comment by combrmendelssohn** [comment]
If they find the solution, which alone exists I still have the solution to the general blocking of scripts

**Comment by innogames-tw2** [comment]
Ok then 😉 even if you find any solution for our changes, we have prepared new update 1.92 😉 with a lot of new "traps". 
If you want a w-w-war, then that's what you'll get 😎. But for now "you shall not pass" 🧙🧙🧙

**Comment by combrmendelssohn** [comment]
I have already found the solution, but I will not pass it on to the public. I have a solution for general blocking. If you want to know about it please contact me. Any solutions you find will be dribbled. there is only one solution to blocking general and I have it.

**Comment by oanimal** [comment]
se a inogames se preocupa se mais a desnvolver coisas interessantes para o jogo funcionar melhor agora a cada actualizacao cada vez tem mais bugs o jogo esta aficar um lixo

**Comment by Siqural** [comment]
And it got updated.

**Comment by BartiemanBro** [comment]
All hail mr Mafrazz! :D

**Comment by Siqural** [comment]
Wondering what did they do actually to make this script not working ;o Did they make it more detectable or what

**Comment by mafrazz** [comment]
Don't mind that innogames account, it's obviously a fake.
Innogames has no intention in create anti-measures about the script for now.

**Comment by Lyrex** [comment]
@mafrazz Okay, I see. If you need any help or something and I'll happily join development.

**Comment by mafrazz** [comment]
Thanks

**Comment by ivanthz** [comment]
Thanks @mafrazz

**Comment by rika86** [comment]
>

> New update?

Yes, inno make an update since from 1.93 to 1.94 since today at midday

and now the script completly stopped working

**Comment by relaxeaza** [comment]
Soon

**Comment by MendelssohnTW** [comment]
Relaxeaza, jQuery está fora.

**Comment by relaxeaza** [comment]
Sim, não tive muito problema com isso, tinha pouca aplicação.
